tests/unit/Core/Content/Media/Infrastructure/Path/MediaUrlGeneratorTest.php

```diff
@@ -27,6 +28,25 @@ public function testGenerate(UrlParams $params, ?string $expected): void
             new Filesystem(new InMemoryFilesystemAdapter(), ['public_url' => 'http://localhost:8000']),
         );
 
+        if (Feature::isActive('v6.8.0.0') && $expected !== null) {
```

Comment:
Also here: the feature check should be in it's own if to make it clear what needs to be removed here. Have you seen the best practices message in slack about deprecations? Might be a good read 🙂 
https://shopware-ag.slack.com/archives/CBD3SQCMC/p1739884109134679